Button/style.css

```diff
@@ -2,6 +2,7 @@
 
 .button {
   display: inline-block;
+  margin: 0;
```

Comment:
It seems like it would be better to move this into `Buffer Web` than here. `button.button` selector perhaps?